include/aos/common/config.hpp

```diff
@@ -879,4 +886,32 @@
 #define AOS_CONFIG_CLOUD_PROTOCOL_BEARER_TOKEN_LEN 256
 #endif
 
+/**
+ * Max subnet length.
+ */
+#ifndef AOS_CONFIG_NETWORKMANAGER_SUBNET_LEN
```

Comment:
These consts are used in types.hpp. So prefix should be TYPES, and should be places where all others types are defined.

Reply:
Done